empirical/util/empirical.py

```
@@ -123,16 +114,20 @@ def get_model(
     try:
         model = classdef.GMM[model_config[tect_type.name][im][component][0]]
     except KeyError:
-        # if rotd50 component is requested and not found, try geom instead. (Do you know why @joel?)
-        if component == constants.Components.crotd50.str_value:
-            try:
-                model = classdef.GMM[
-                    model_config[tect_type.name][im][
-                        constants.Components.cgeom.str_value
-                    ][0]
-                ]
-            except KeyError:
-                pass
+        if fallback_comp is not None:
+            # if rotd50 component is requested and not found, try geom instead. (Do you know why @joel?)
+            if component == constants.Components.crotd50.str_value:
+                try:
+                    model = classdef.GMM[
+                        model_config[tect_type.name][im][fallback_comp][0]
+                    ]
+                except KeyError:
+                    pass
+
+        else:
+            print(
+                f"Warning: Model not found for {tect_type.name}, {im}, {component}. Skipping..."
+            )
```

Something doesn't quite seem right here, I like the idea of a fallback component, but I think it should just try lines 120 - 125 and ignore the extra rotd50 check.

Again another option would be just to have in the config file both models for each of rodt50 and geom mentioned and then if the component is not found on first try then just skip.

---

I also felt like to remove the rotd50 check - only kept it as I was unsure if there was any special reason attached to rotd50. I'm happy to remove it, and even took a further step. I actually prefer to raise an error instead of silently switching to a fallback component.